docs/general-settings.md

```diff
@@ -56,6 +56,15 @@ settings: {
 }
 ```
 
+## Parameters
+
+Below is a list of the parameters you can use in the `settings:` Luigi configuration section.
```

Comment:
```suggestion
Below a list of the parameters you can use in the `settings:` Luigi configuration section.
```

Reply:
In this case I think it's best to leave the "is", as it's the only verb in the sentence and removing it would make it incomplete: https://examples.yourdictionary.com/reference/examples/examples-of-complete-sentences.html